Library/ViewModels/DiscoveryPostcardViewModel.swift

```diff
@@ -217,11 +230,26 @@ public final class DiscoveryPostcardViewModel: DiscoveryPostcardViewModelType,
   }
   // swiftlint:enable function_body_length
 
+  fileprivate let delegateDidSetProperty = MutableProperty()
+  public func delegateDidSet() {
+    self.delegateDidSetProperty.value = ()
+  }
```

Comment:
i know brando already pointed out you aren't using this property in another place, but just marking this bit of code to eliminate as well.